bin/generate-css-utility-classes-docs.js

```diff
@@ -43,4 +187,20 @@ const stylesheetsDirectory = path.resolve(
   } catch (error) {
     throw new Error('Error generating the CSS utilty class Storybook stories');
   }
-})();
+}
+
+if (process.env.NODE_ENV !== 'test') {
+  // we're running unit tests so do not run the function to generate docs.
+  generateDocumentation();
+}
+
+/*
+  These variables and functions are only used within this script, but they are being exported
+  so that the core of the doc generation tool can be tested. Normally you would not export
+  things deemed private.
+*/
+module.exports = {
```

Comment:
Last question before me merge @ben, @citizen428 and @rhymes. I export these as mentioned in the script just to test codegen core functionality. The tests are co-located in `__tests__` in the `bin` folder. Are y'all OK with a `__tests__` folder in the `bin`? If not I can move it to some folder under `app/javascript`.